### PR TITLE
add colt 1911 ammo design disk to cargo ruin

### PR DESCRIPTION
**Pull request by Maldaris**
## About The Pull Request

After #439 , 1911 Ammo wasn't obtainable by any method available to the player. This alleviates that in a way that makes the ruin where 1911s come from more worthwhile to clear.

## Why It's Good For The Game

Fudd gun needs its bullets

## Changelog
:cl:
add: A 1911 Ammo Disk for use in an autolathe, in the same ruin that 1911s spawn in.
/:cl:

### CODE REVIEW

**Comment by ADepressedPyroshark** [comment]
Ignoring the funny empty pr, is there anything letting the player know that Miss Tiggles drops it? Maybe a "In their dying breath, Miss Tiggles Vomits up a Design Disk!" kinda of thing? It would be better than it just popping out with no context. Also i'm somewhat afraid it will hide under the corpse or something.

**Comment by Maldaris** [comment]
Note to self: Don't PR while you're half asleep.

To your point Pyro, I'd like to see the existing behavior first. The property I extended to drop the disk is core to the `simple_animal` code.

File: waspstation/code/modules/projectiles/boxes_magazines/pistol.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/datum/design/ammo/colt_1911_magazine
+	name = "Colt 1911 Magazine"
+	id = "ammo_1911"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron = 4000)
+	build_path = /obj/item/ammo_box/magazine/m45
+	category = list("Security")
```
Comment (Superyodeler) [inline waspstation/code/modules/projectiles/boxes_magazines/pistol.dm:7]: All other design disks add designs to the Imported catagory.
```suggestion
	category = list("Imported")
```

File: waspstation/code/modules/projectiles/boxes_magazines/pistol.dm [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+/obj/item/disk/design_disk/ammo_1911
+	name = "Design Disk - 1911 Magazine"
+	desc = "A design disk containing the pattern for the classic 1911's seven round .45ACP magazine."
+	var/ammo_design = /datum/design/ammo/colt_1911_magazine
+
+/obj/item/disk/design_disk/ammo_1911/Initialize()
+	. = ..()
+	blueprints[1] = ammo_design
```
Comment (Superyodeler) [inline waspstation/code/modules/projectiles/boxes_magazines/pistol.dm:16]: Changed to the method used by other disks.
```suggestion
/obj/item/disk/design_disk/ammo_1911
	name = "Design Disk - 1911 Magazine"
	desc = "A design disk containing the pattern for the classic 1911's seven round .45ACP magazine."

/obj/item/disk/design_disk/ammo_1911/Initialize()
	. = ..()
	var/datum/design/colt_1911_magazine/M = new
	blueprints[1] = M
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Adding the design to the autolathe currently runtimes and doesn't work.

File: waspstation/code/modules/projectiles/boxes_magazines/pistol.dm [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	build_path = /obj/item/ammo_box/magazine/m45
+	category = list("Imported")
+
+/obj/item/disk/design_disk/ammo_1911
+	name = "Design Disk - 1911 Magazine"
+	desc = "A design disk containing the pattern for the classic 1911's seven round .45ACP magazine."
+
+/obj/item/disk/design_disk/ammo_1911/Initialize()
+	. = ..()
+	var/datum/design/colt_1911_magazine/M = new
```
Comment (MarkSuckerberg) [inline waspstation/code/modules/projectiles/boxes_magazines/pistol.dm:15]:
```suggestion
	var/datum/design/ammo/colt_1911_magazine/M = new
```

**Comment by Maldaris** [comment]
@Superyodeler if you can sign off on this that'd be great.

**Comment by Superyodeler** [comment]
I will check this out again today or tomorrow.

**Review by Superyodeler — APPROVED** [review]

**Comment by Maldaris** [comment]
Passed vote, merging.

**Pull request merged**